packages/fast-components-react-msft/src/action-toggle/action-toggle.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Action toggle",
+    "description": "An action toggle component's schema definition.",
+    "type": "object",
+    "id": "action-toggle",
+    "properties": {
+        "disabled": {
+            "title": "Disabled",
+            "type": "boolean"
+        },
+        "selected": {
+            "title": "Selected",
+            "type": "boolean"
+        },
+        "selectedContent": {
+            "title": "Selected text",
+            "type": "object"
+        },
+        "unselectedContent": {
+            "title": "Unselected text",
+            "type": "object"
```

Comment:
These are not objects in the sense that you are not enumerating properties on a regular object, they are instead ReactNodes, now if they are not react renderprops you should use place these in "reactProperties" with the type set as "children" see: https://github.com/Microsoft/fast-dna/tree/master/packages/fast-form-generator-react#react-children

Looks like you added the property below, just need to place these there with the appropriate type set.

Reply:
worked, thx

Reply:
So shouldn't it be removed here now?

Reply:
Updated with changes.